tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -1475,7 +1475,7 @@ private long[] setBestNextAndSameVoteValues(Set<T> sortedValues, HashMap<T, Long
             ++i;
             long valueWeight = voteCount.get(value);
             if (i == 0) {
-                winningValue = value;
+                setWinningValue(value);
```

Comment:
this is the typical case when there are enough votes -- it was already the case that we would return inheritance marker here rather than bailey

when setWinningValue is called from here, changeBaileyToInheritance in setWinningValue should always be superfluous, since that's already done before vote resolution -- nevertheless we always use the setter for encapsulation

Reply:
thanks for updating the transcript annotations!